pkg/prometheus/promcfg_test.go

```diff
@@ -6837,6 +6837,54 @@ func TestScrapeConfigSpecConfigWithConsulSD(t *testing.T) {
 				},
 			},
 			golden: "ConsulScrapeConfig.golden",
+		}, {
+			name:    "consul_scrape_config_with_filter",
```

Comment:
can we modify the test to only have required fields and then new `filter` field so test will make it clear?

Reply:
Thanks @slashpai 
I got it. Fixed